riscv/src/small_field/runtime.rs

```diff
@@ -272,8 +274,15 @@ impl Runtime {
         }
     }
 
-    fn with_poseidon(self, _continuations: bool) -> Self {
-        todo!()
+    // TODO This implementation is just a placeholder,
+    // needed by the runtime "finalize" code.
+    fn with_poseidon(mut self, continuations: bool) -> Self {
+        assert!(!continuations);
+
+        let implementation = std::iter::once("affine 0, 0, 0, 0, 0, 0;".to_string());
```

Comment:
This could be a no-op, but the submachine mechanism checks that this parses as a `FunctionStatement`.